src/jit/flowgraph.cpp

```diff
@@ -13146,68 +13160,96 @@ void Compiler::fgComputeEdgeWeights()
 #if DEBUG
     if (verbose && modified)
     {
-        printf("fgComputeEdgeWeights() adjusted the weight of some blocks\n");
+        printf("fgComputeMissingBlockWeights() adjusted the weight of some blocks\n");
         fgDispBasicBlocks();
         printf("\n");
     }
 #endif
 
+    return returnWeight;
+}
+
+//-------------------------------------------------------------
+// fgComputeCalledCount: if profile information is in use,
```

Comment:
Nit: this seems to imply that this will behave conditionally on whether profile information is in use, but it seems that it is now assumed, so this should probably say "Compute fgCalledCount when profile information is in use."